Run3Detector/analysis/utilities/milliqanCuts.py

```diff
@@ -6,6 +6,30 @@ class milliqanCuts():
 
     def __init__(self):
         self.events = []
+        self.cutflow = []
+        self.counter = 0
+
+    def cutflowCounter(self):
+        # Increments events passing each stage of the cutflow
+        # Creates each stage during the first pass
+        try:
+            self.cutflow[self.counter]+=len(self.events)
+        # Builds the array without knowledge of the number of cuts
+        # TODO: See if there is a more efficent way to implement this
+        except:
+            self.cutflow.append(len(self.events))
+        self.counter+=1
+
+    def getCutflowCounts(self):
+        # Prints the value after each batch of events
+        # TODO: Only print at the very end
+        print(self.cutflow)
```

Comment:
remove this print before the actual cutflow printing?

Reply:
Done